python_modules/dagster/dagster/_components/__init__.py

```diff
@@ -227,7 +228,7 @@ def __repr__(self) -> str:
 
 
 class ComponentLoadContext:
-    def __init__(self, resources: Mapping[str, object] = {}):
+    def __init__(self, resources: Mapping[str, object]):
```

Comment:
https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python